contrib/core/actions/send_mail/send_mail

```diff
@@ -52,7 +52,7 @@ if [[ -z $trimmed && $SEND_EMPTY_BODY -eq 1 ]] || [[ -n $trimmed ]]; then
 cat <<EOF
 TO: ${TO}
 FROM: ${FROM}
-SUBJECT: ${SUBJECT}
+SUBJECT: =?UTF-8?q?${SUBJECT}?=
```

Comment:
I assume this works fine with all content types - text/html (default) and text/plain?

Reply:
If I understand it correctly based on this, https://tools.ietf.org/html/rfc1342, the encoding of the header is separate from the encoding of the message (which is why this needs to be specified separately). I've tested `text/plain` and 'text/html' and both seem to work fine.